packages/scripts/webapp/preset/config/webpack.config.prod.js

```diff
@@ -12,6 +12,22 @@ module.exports = ({ getUserConfig }) => {
 	return {
 		mode: 'production',
 		optimization: {
+			/**
+			 * Let’s create a vendor chunk which contain all code from
+			 * node_modules imported in the project.
+			 * Whenever, Webpack encounters import statement of npm module,
+			 * it will push that code inside vendor chunk.
+			 */
+			splitChunks: {
+				cacheGroups: {
+					default: false,
+					vendors: false,
+					vendor: {
```

Comment:
Do you find any good resource on that point? I'm working on the same topic on my side.